cmd/node/node.go

```diff
@@ -1,5 +1,6 @@
 package node
 
+import "C"
```

Comment:
Hey man, can you explain this `import`? It causes our entire project to use `cgo` and breaks cross-compilation. I also don't think it's used anywhere...